src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import { func } from 'prop-types';
+import { string } from 'prop-types';
+
+function Button(props) {
+  return (
+    <button
+      className={'button button--' + (props.size ? props.size : 'medium')}
+      onClick={props.onClick}
```

Comment:
you can add {...props} to the <button> tag to be able to add additional things related to button (like `type`, for example)